src/utils/siteColorUtils.js

```diff
@@ -63,7 +63,7 @@ const createPageStyleSheet = (repoName, primaryColor, secondaryColor) => {
 
   // EditHomepage: hero button - secondary color
   customStyleSheet.insertRule(
-    `.is-secondary { background-color: ${secondaryColor} !important;}`,
+    `.is-secondary { background-color: ${secondaryColor} !important; color: #fff !important;}`,
```

Comment:
can i check where you got this from?

Reply:
![Screenshot 2023-11-09 at 1 06 21 PM](https://github.com/isomerpages/isomercms-frontend/assets/42832651/3468d8f8-ebe7-460c-95bb-a3aec9caf997)
hmm am worried about regression too, got it from this line here. is this sound?

Reply:
We should check the scss file itself to see the condition. 

This also requires an additional class `bp-button` so the conditions aren't exactly the same

Reply:
to be clear thats from the live site, the one in our preview is here!
![Screenshot 2023-11-10 at 4 20 51 PM](https://github.com/isomerpages/isomercms-frontend/assets/42832651/ccd4a6a5-6313-4620-a912-cd8e94486019)